### ISSUE TITLE
Independent volume control

### ISSUE THREAD

**Issue by Lantern1984** [Genymobile/scrcpy#4826]
- [✔ ] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

Is there some way to control the volume of the window without having to turn down the phone or computer's volume? I like to listen to audio books from my phone, but they come up too loud when I use the device or computer speakers. I have tried turning down the volume of just the window in volume mixer, but every time I have to reopen scrcpy, the volume is back at max. I like to leave the phone's volume alone because it is just right for making phone calls, and the computer's master volume needs to be kept high for certain alerts, so I need a third option.

**Comment by 18637** [comment]
Yes. You should be able to control the audio from Scrcpy independently. Go to windows audio control and you should be able to adjust the scrcpy audio without changing the volume of the computer (If you are on windows)
